packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -731,8 +731,6 @@ class ThemeData extends Diagnosticable {
   /// Linearly interpolate between two themes.
   ///
   /// {@template flutter.material.themeData.lerp}
-  /// The arguments must not be null.
-  ///
```

Comment:
Why did you remove this statement? It's still true for this method.

Reply:
looks like it was just an oversight

Reply:
Yes sorry it was a mistake, I meant to move it outside of the template definition.  Will fix.